### PR TITLE
Style guide

### PR DESCRIPTION
**Pull request by CheezItMan**
This is an introduction to running JavaScript locally via Node.  We can drop this if y'all think it's inappropriate, but I thought it might be useful to take away the magic C15 found in working with Create-React-App and node.

Content:

1.  How to get node running
2. VS Code extensions (prettier and eslint)
3. Code Style and Style Guides
4. How to set up and run a node project

If the style rules look ok, I'd like to add'em to our JS projects as well.

### CODE REVIEW

**Review by spitsfire — APPROVED** [review]
I am so down for this! I think it makes sense we show them how to do it locally since we did that with python as well

File: learning-another-language/vscode-and-style.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+## Style Guides
+
+![How Coding Standards Proliferate](../assets/learning-another-language__vscode-and-style__coding-stanards.png)
+
+*Source: [XKCD](https://xkcd.com/927/)*
+
+Many software teams adhere to a set of rules which serve as guidelines for how to write and organize code. Linters can be an integral part of a team's style conventions.
+
+**Why use a style guide?**  Style guides are a common way for a group of people to create consistently formatted documents. Like MLA or the Chicago Manual of Style, a code style guide helps us find the information we are looking for as quickly as possible.
```
Comment (kaidamasaki) [inline learning-another-language/vscode-and-style.md:118]: Consider making this a heading:

```suggestion
### Why use a style guide?

Style guides are a common way for a group of people to create consistently formatted documents. Like MLA or the Chicago Manual of Style, a code style guide helps us find the information we are looking for as quickly as possible.
```

**Review by kaidamasaki — COMMENTED** [review]
This seems reasonable, provided we don't make them write a project in Node again.  (Dealing with IO is _really_ awful.  >< )

**Comment by CheezItMan** [comment]
@kaidamasaki I suppose we could have them work with `fs` (Filesystem) for input/output.

**Pull request merged**